Out of jokes changelog update (#48567)
CleverRaven/Cataclysm-DDA: data/changelog.txt

Fixes crashes from canceling reloading a container that is on the ground.
Fixes NPCs still being hungry - and unhappy about it - after eating from faction camp stores.

> Reined in plant mutation photosynthesis ability to be more representative of real photosynthesis.

Reviewer: ```suggestion
Reins in the plant mutation category's photosynthesis ability to be more representative of real photosynthesis.
```

> Prevents cross-z-level melee attacks when attacker or target is not actually visible.
> Do not do character body updates more than once per turn
> When harvesting crops, drop the result on player's square.
> Improves road system connectivity across Overmaps.

Reviewer: ```suggestion
Prevents cross-z-level melee attacks when the target is not actually visible.
Avoids doing character body updates more than once per turn
When harvesting crops, drops the result on the player's square.
Improves road system connectivity across overmaps.
```

Reduces file size of composed tilesheets from compose.py.

> Implements C++ symbol demangling in crash reports on MacOS, MinGW, and Linux.

Reviewer 2: ```suggestion
 Implements C++ symbol demangling in crash reports on MinGW, Linux, macOS, BSD and Android.
```

Reviewer: ```suggestion
Implements C++ symbol demangling for crash reports on MinGW, Linux, macOS, BSD, and Android.
```
